### PR TITLE
Feature/m0078 ddg 406 migrate sdk release notes

### PR DESCRIPTION
**Pull request by frivieremicroej**

### CODE REVIEW

File: overview/sdkChangeLog.rst [hunk 1]
```diff
@@ -0,0 +1,895 @@
+MicroEJ SDK Distribution Changelog
+----------------------------------
+
+[21.03] - 2021-03-25
+~~~~~~~~~~~~~~~~~~~~
+
+-  Included MicroEJ Studio / SDK ``5.4.0``
```
Comment (frivieremicroej) [inline overview/sdkChangeLog.rst:7]: Please turn all SDK version to clickable link to the changelog section (everywhere):
Example:
image

File: overview/sdkChangeLog.rst [hunk 2]
```diff
@@ -0,0 +1,895 @@
...
+Skeletons
+^^^^^^^^^
+
+-  ``[Studio]`` Fixed wrong ``README.md`` generation for
+   ``artifact-repository`` skeleton
+-  ``[SDK]`` Removed useless files in ``microej-javaapi``,
+   ``microej-javaimpl`` and ``microej-extension`` skeletons (intern
+   changelog and ``.dbk`` file)
+
+Known Issues
```
Comment (frivieremicroej) [inline overview/sdkChangeLog.rst:192]: Known issue / misc wrong title level (shoudl not appear in the toc)
image

File: overview/sdkChangeLog.rst [hunk 3]
```diff
@@ -0,0 +1,895 @@
+MicroEJ SDK Distribution Changelog
```
Comment (frivieremicroej) [inline overview/sdkChangeLog.rst:1]: changelog sections shall be moved under release notes
image

**Comment by frivieremicroej** [comment]
Preview: https://docs.microej.com/en/feature-m0078ddg-406_migrate_sdk_release_notes/overview/sdk.html

File: overview/sdk.rst [hunk 4]
```diff
@@ -45,8 +48,17 @@ detailed in the following sections of this chapter :
 
 -  :ref:`Build and run your first Application on Device <device_build>`
 
+.. include:: sdkReleaseNotes.rst
+
+.. include:: sdkChangeLog.rst
+
+.. include:: sdkInstallationNotes.rst
+
+.. include:: sdkMigrationNotes.rst
```
Comment (tdelhomenie-microej) [inline overview/sdk.rst:57]: We should not include the content of these pages in the SDK Overview page, it makes the page really big for an overview. And most of the time when I go to an overview page of a product I don't need the details of the content of each release. Most of the products do it this way.
I would just add links for each one of these inclusions and make them independent pages.

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: overview/sdk.rst (hunk 4 above)
Comment (frivieremicroej) [reply overview/sdk.rst]: thanks it's ok now

**Pull request merged**